packages/models-library/src/models_library/service_settings_nat_rule.py

```diff
@@ -1,21 +1,14 @@
 from collections.abc import Generator
 from typing import Final
 
-from pydantic import (
-    BaseModel,
-    ConfigDict,
-    Field,
-    ValidationInfo,
-    field_validator,
-    parse_obj_as,
-)
+from pydantic import BaseModel, ConfigDict, Field, ValidationInfo, field_validator
 
 from .basic_types import PortInt
 from .osparc_variable_identifier import OsparcVariableIdentifier, raise_if_unresolved
 
 # Cloudflare DNS server address
 DEFAULT_DNS_SERVER_ADDRESS: Final[str] = "1.1.1.1"  # NOSONAR
-DEFAULT_DNS_SERVER_PORT: Final[PortInt] = parse_obj_as(PortInt, 53)
+DEFAULT_DNS_SERVER_PORT: Final[PortInt] = 53
```

Comment:
Adapter that validates PortInt? AFAI remember, PortInt validates only on a positive range of integers